jbpm-wb-process-runtime/jbpm-wb-process-runtime-backend/src/test/java/org/jbpm/workbench/pr/backend/server/util/VariableHelperTest.java

```diff
@@ -91,4 +92,23 @@ public void addDeploymentIdServerTemplateToAllProcessInstancesVarsTest() {
         assertEquals("", varList.get(2).getNewValue());
         assertEquals("", varList.get(3).getNewValue());
     }
+
+    @Test
+    public void sortProcessInstancesVarsByNameTest() {
+        List<ProcessVariableSummary> varList =
+                VariableHelper.adaptCollection(variables, properties, processInstanceId, deploymentId, serverTemplateId,
+                                               COL_PROCESS_INSTANCE_VAR_ID, false);
+        assertEquals(4, varList.size());
+        assertEquals(processDefaultVar_name, varList.get(0).getName());
```

Comment:
The sort order seems to be really reversed .... the variable is called asc, and when false, we're happy with values being in ... ascending order.